converter/dicom-cast/src/Microsoft.Health.DicomCast.Core/Features/Worker/FhirTransaction/ImagingStudy/ImagingStudyInstancePropertySynchronizer.cs

```diff
@@ -26,14 +44,54 @@ public void Synchronize(FhirTransactionContext context, ImagingStudy.InstanceCom
                 return;
             }
 
-            // Add sopclass to instance
+            SynchronizePropertiesAsync(instance, context, true, AddSopClass, cancellationToken);
+            SynchronizePropertiesAsync(instance, context, false, AddInstanceNumber, cancellationToken);
```

Comment:
is it ok to add parameter names when calling SynchronizePropertiesAsync?
the meaning of true/false and AddInstanceNumber/AddSopClass is not very explicit when reading method invocation.

Reply:
I made the argument names more clear so hopefully it shouldnt be an issue anymore